module02/ex03/ex03.md

```diff
@@ -60,7 +62,7 @@ You can make the class skip lines at the top and the bottom of the file,
 and also keep the first line as a header if `header` is `True`.
 
 The file should not be corrupted (either a line with too many values or a line with too few values), otherwise return `None`.
-You have to handle the case `file not found`.
+You have also to handle the case `file not found`.
```

Comment:
have also -> also have

Reply:
not exactly the same sentence on the subject intended for holygraph (You have to handle the case \texttt{file not found})